Quantum-Control-Applications/Superconducting/Single Fixed Transmon/ramsey.py

```diff
@@ -28,9 +29,9 @@
     with for_(n, 0, n < n_avg, n + 1):
         # Notice it's <= to include t_max (This is only for integers!)
         with for_(tau, tau_min, tau <= tau_max, tau + dtau):
-            play("pi_half", "qubit")
+            # play("x90", "qubit")
```

Comment:
Why is this commented?

Reply:
I was testing something and forgot to uncomment